docs/packaging/packaging-practices.md

```diff
@@ -172,7 +164,7 @@ pkgconfig(gtk+-wayland-3.0) pkgconfig(gtk+-x11-3.0)
 
 ### Using pkgconfig dependencies
 
-In the `builddeps` list, simply use the `pkgconfig(name)` syntax. For example, to add gtk+-3.0 to the build dependencies, we would do the following:
+In the `builddeps` list, simply use the `pkgconfig(name)` syntax. For example, to add `gtk+-3.0` to the build dependencies, you would do the following:
```

Comment:
```suggestion
In the `builddeps` list, use the `pkgconfig(name)` syntax. For example, to add `gtk+-3.0` to the build dependencies, you would do the following:
```

"Simply" used this way grinds my gears. Delete on sight

Reply:
Good point